<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1217</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Робота з даними.</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Меню</string>
       </attribute>
       <widget class="QPushButton" name="read_file_button_1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>161</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Відкрити файл</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="data_table_widget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>521</width>
          <height>441</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Статистики</string>
       </attribute>
       <widget class="QWidget" name="plot_widget_1" native="true">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>611</width>
          <height>461</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableWidget" name="data_table_widget_2">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>70</y>
          <width>491</width>
          <height>341</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
